test/over_react/component_declaration/component_base_test.dart

```diff
@@ -30,56 +30,103 @@ import '../../test_util/test_util.dart';
 import '../shared/map_proxy_tests.dart';
 
 main() {
-  group('component base:', () {
-    group('UiProps', () {
-      group('warns and throws when rendering a DOM component', () {
-        bool warningsWereEnabled;
-        setUp(() {
-          warningsWereEnabled = ValidationUtil.WARNINGS_ENABLED;
-          ValidationUtil.WARNINGS_ENABLED = false;
-          startRecordingValidationWarnings();
-        });
+  void _commonNonInvokedBuilderTests(UiProps factory) {
```

Comment:
These arguments should be named `builder` to be consistent with our variable naming conventions:
- `UiFactory` - `factory`
- `UiProps` - `builder` / `props`